Sources/MapboxMaps/Style/Style.swift

```diff
@@ -277,10 +260,21 @@ public class Style {
 
 // See `StyleManagerProtocol` for documentation for the following APIs
```

Comment:
Looks like this might have been accidentally left in when address this feedback https://github.com/mapbox/mapbox-maps-ios/pull/708#issuecomment-926690674

Reply:
Good catch

Reply:
Fixed by https://github.com/mapbox/mapbox-maps-ios/pull/710